docs/getting-started-guides/docker-multinode.md

```diff
@@ -53,7 +49,9 @@ Please install Docker 1.6.2 or Docker 1.7.1.
 
 ## Prerequisites
 
-1. You need a machine with docker of right version installed.
+The only thing you need is a machine with **Docker 1.8.x or higher**
+
+It might work with older docker versions, but that isn't supported.
```

Comment:
We do use `docker daemon` differently. The issue is about combining `docker -d --bridge=none` and `docker run --net=host`.

We do this for `etcd` and `flannel`, where they have `--net=host` and `docker-bootstrap` has `--bridge=none`.

I don't really know what to do to make everyone happy.
I assume @brendandburns wants docker >= 1.8.x because he said it, but should I make it like this?
> It _should_ work with older docker versions (like 1.7.1), but that isn't fully supported.